app/main.py

```diff
@@ -16,17 +16,12 @@ def is_list_of_strs(input):
 
     def prediction(self, input):
         if 'words' not in input:
```

Comment:
We should also put a limit on the number of words that can be in the input. If we dont someone could send us a whole dictionary and kill the provider.

Reply:
Good idea, I will add that tonight.